next.config.js

```diff
@@ -1,6 +1,14 @@
 // next.config.js
 module.exports = {
   images: {
-    domains: ["picsum.photos", "dummyimage.com"],
+    domains: [
+      "picsum.photos",
+      "dummyimage.com",
+      "images.unsplash.com",
+      "insidethemagic-119e2.kxcdn.com",
+      "saltinmycoffee.com",
+      "breedingbusiness.com",
+      "unsplash.com",
+    ],
```

Comment:
My first thought on this is that I wonder if this can be done dynamically. Really not sure if that's possible so it'll be worth having a read of the documentation. My understanding is that these domains determine where an image can be hosted to be used in the `next/Image` component so I'm just wondering what would happen if an admin user uploaded an image that was hosted somewhere different.


This might not even be a possibility, in which case you can ignore this whole comment.

Reply:
I thought about it too, but it seems like it's not possible yet. (or maybe I just missed something)

Reply:
check the comments here: 
https://github.com/vercel/next.js/discussions/18311

Reply:
https://github.com/vercel/next.js/pull/27345

Reply:
As discussed yesterday, think you could just use normal <image> tags for anything uploaded by the user instead of trying to mess around with this.